src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl

```diff
@@ -38565,6 +38565,7 @@ unit:PER-K
   qudt:hasQuantityKind quantitykind:LinearExpansionCoefficient ;
   qudt:hasQuantityKind quantitykind:RelativePressureCoefficient ;
   qudt:hasQuantityKind quantitykind:ThermalExpansionCoefficient ;
+  qudt:hasQuantityKind quantitykind:apparentThermalInertia .
```

Comment:
Should be a ; not a .

Reply:
fixed in 15d1360fefa7f2bec2ba499ee6f4f217a005a85e